### PR TITLE
fix baggage contains null

### PR DESCRIPTION
**Pull request by OrezzerO**
### Motivation:

If baggage contains null value, `putAll` method of. ConcurrentHashMap will throw NPE.

### CODE REVIEW

File: core/api/src/main/java/com/alipay/sofa/rpc/context/RpcInvokeContext.java [hunk 1]
```diff
@@ -226,6 +226,7 @@ public Object remove(String key) {
      */
     public void putRequestBaggage(String key, String value) {
         if (BAGGAGE_ENABLE && key != null && value != null) {
+
```
Comment (JervyShi) [inline core/api/src/main/java/com/alipay/sofa/rpc/context/RpcInvokeContext.java:229]: This line of code is unnecessary.

Comment (OrezzerO) [reply core/api/src/main/java/com/alipay/sofa/rpc/context/RpcInvokeContext.java]: Deleted.

**Review by chuailiwu — APPROVED** [review]
LGTM

**Review by EvenLjj — APPROVED** [review]
LGTM

**Pull request merged**